src/content/interop/js-interop/js-types.md

```diff
@@ -252,4 +275,8 @@ TODO: add links (with stable) when ready:
 [`instanceOfString`]: https://api.dart.dev/dev/dart-js_interop/JSAnyUtilityExtension/instanceOfString.html
 [`isA`]: https://api.dart.dev/dev/dart-js_interop/JSAnyUtilityExtension/isA.html
 [#4841]: https://github.com/dart-lang/linter/issues/4841
-[#54025]: https://github.com/dart-lang/sdk/issues/54025
\ No newline at end of file
+[#54025]: https://github.com/dart-lang/sdk/issues/54025
+[`JSBoxedDartObject`]: https://api.dart.dev/main/dart-js_interop/JSBoxedDartObject-extension-type.html
+[`ExternalDartReference`]: https://api.dart.dev/main/dart-js_interop/ExternalDartReference-extension-type.html
+[`toExternalReference`]: https://api.dart.dev/main/dart-js_interop/ObjectToExternalDartReference/toExternalReference.html
+[`toDartObject`]: https://api.dart.dev/main/dart-js_interop/ExternalDartReferenceToObject/toDartObject.html
```

Comment:
```suggestion
[`JSBoxedDartObject`]:  {{site.dart-api}}/{{site.sdkInfo.channel}}/dart-js_interop/JSBoxedDartObject-extension-type.html
[`ExternalDartReference`]:  {{site.dart-api}}/{{site.sdkInfo.channel}}/dart-js_interop/ExternalDartReference-extension-type.html
[`toExternalReference`]:  {{site.dart-api}}/{{site.sdkInfo.channel}}/dart-js_interop/ObjectToExternalDartReference/toExternalReference.html
[`toDartObject`]:  {{site.dart-api}}/{{site.sdkInfo.channel}}/dart-js_interop/ExternalDartReferenceToObject/toDartObject.html
```

The rest of the links were updated in [this PR](https://github.com/dart-lang/site-www/pull/5667/files), so I think you'll get those changes if you just update the branch. This suggestion is to correct the new ones.

Reply:
Done, thanks!